components/ai_chat/core/engine/engine_consumer_claude.cc

```diff
@@ -194,8 +196,10 @@ void EngineConsumerClaudeRemote::GenerateAssistantResponse(
     const std::string& human_input,
     GenerationDataCallback data_received_callback,
     GenerationCompletedCallback completed_callback) {
-  std::string prompt = BuildClaudePrompt(human_input, page_content, is_video,
-                                         conversation_history);
+  const std::string& truncated_page_content =
```

Comment:
This only applies to conversation prompts, but not other prompts. At the moment, the other prompt generated is `GenerateQuestionSuggestions`, so you'll need to also truncate when building that prompt. ditto for llama

Reply:
done